packages/curve-ui-kit/src/themes/mui-overrides.d.ts

```diff
@@ -17,3 +17,9 @@ declare module '@mui/material/styles' {
     highlight: string
   }
 }
+
+declare module '@mui/material/CardHeader' {
```

Comment:
If you decide to create a `defineMuiCard()` file, function and folder this could be moved to `override-cards.d.ts`

Reply:
I want to reorganize the mui folder once we don't have open PRs in this area. It's not really consisten rn